### ISSUE TITLE
configuration lost when Heroku dynos restart

### ISSUE THREAD

**Issue by lnhrdt** [dz0ny/heroku-docker-piwik#1]
When dynos restart the app looses configuration because it's saving configuration to the filesystem (default behavior). Piwik has a configuration option to save everything to the DB by setting:

```
session_save_handler = dbtable
```

http://piwik.org/faq/how-to-install/faq_133/

Any chance this configuration can be built into this package?

**Comment by dz0ny** [comment]
That's only session data, you still need to persist config file. The easiest way is to add it to repo once you created project(security wise this is not good). Another way is to populate config file during "heroku compile". 

TLDR; Working on it :)

**Comment by Floppy** [comment]
@dz0ny sorry to bump an old issue, but I'm having exactly this problem with a deploy I only created a few days ago - all the environment variables are set, and I followed all the instructions as far as I can tell. It's like the replace isn't happening in config.ini, but I've no idea what I've missed that means that's not happening. I'm running a hobby dyno on Heroku with Jaws MariaDB, and it all works fine apart from losing config on dyno restart. Any ideas?

**Comment by jasonfb** [comment]
I too am confused about where the intended settings for the database goes — I have this working with a remote database (in my case it happens to be Amazon RDS), I can get through the setup, but upon restarting the dynos it reverts to the original settings. I've tried modifying ```config.ini.php```, and removing line 1, each time I restart the dynos the app reverts to the start-up screen.

I read some comments about creating an over-ride file at vendor/piwik/piwik/config.ini.php

I went to attempt this, but ```vendor/``` is in the ```.gitignore```, so I think I may misunderstand something. Is the creating of the override file something that happens inside of the Heroku? 

I have specified DB_HOST, DB_NAME, DB_PASSWORD, DB_PORT, DB_USERNAME as config vars on my heroku app, but it seems to not be able to pick those up.
